core/cmd/jobs_commands.go

```diff
@@ -106,6 +106,10 @@ func (p JobPresenter) FriendlyCreatedAt() string {
 		if p.KeeperSpec != nil {
 			return p.KeeperSpec.CreatedAt.Format(time.RFC3339)
 		}
+	case presenters.CronJobSpec:
```

Comment:
cc @spooktheducks this seemed to be missing

Reply:
test?

Reply:
added